stories/assets/scss/_variables.scss

```diff
@@ -73,10 +73,10 @@ $spacing-160: 10rem;
 $width-600: 37.5rem;
 
 // GRADIENT COLOR LIST
-$yellow-gradient: $color-yellow 1.21%, $color-yellow 56.78%, rgba(255, 235, 0, 0.7) 100%;
-$red-gradient: $color-red 1.21%, $color-red 56.78%, rgb(240 49 49 / 70%) 100%;
-$green-gradient: $color-green 1.21%, $color-green 56.78%, rgb(83 230 80 / 70%) 100%;
-$blue-gradient: $color-azure 1.21%, $color-azure 56.78%, rgb(101 204 244 / 70%) 100%;
+$yellow-gradient: $color-yellow 30%, rgba(255, 235, 0, 0.2) 110%;
+$red-gradient: $color-red 30%, rgb(240, 49, 49, 0.2) 110%;
+$green-gradient: $color-green 30%, rgba(0, 0, 0, 0.2) 110%;
+$blue-gradient: $color-azure 30%, rgb(101, 204, 244, 0.2) 110%;
```

Comment:
@Shamimkhan321 We are changing this but can you check if this change will impact other places?